### ISSUE TITLE
Drawers don't handle deleted items gracefully

### ISSUE THREAD

**Issue by gchpaco** [jaquadro/StorageDrawers#56]
I accidentally had an instance with Magical Crops installed; after a while of storing some of the output in drawers, I decided to remove the mod.  WAILA shows "\<ERROR\>" when looking at the drawers that used to hold the stuff, and I get tracebacks like this whenever I interact with those drawers.

Interestingly re-adding Magical Crops doesn't help.

```
Time: 4/7/15 3:31 AM
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
        at com.jaquadro.minecraft.storagedrawers.block.tile.TileEntityDrawers.pu
tItemsIntoSlot(TileEntityDrawers.java:160)
        at com.jaquadro.minecraft.storagedrawers.block.tile.TileEntityDrawers.in
teractPutCurrentItemIntoSlot(TileEntityDrawers.java:178)
        at com.jaquadro.minecraft.storagedrawers.block.tile.TileEntityDrawers.in
teractPutItemsIntoSlot(TileEntityDrawers.java:209)
        at com.jaquadro.minecraft.storagedrawers.block.BlockDrawers.func_149727_
a(BlockDrawers.java:252)
        at net.minecraft.client.multiplayer.PlayerControllerMP.func_78760_a(Play
erControllerMP.java:326)
        at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1435)
        at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1941)
        at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961)
        at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
```

**Comment by gchpaco** [comment]
I suppose I should say expected behavior is "deleted items vanish and drawer remains useful".

**Comment by jaquadro** [comment]
This should be fixed in 1.3.7.  The items should quietly vanish during chunk load.

**Comment by gchpaco** [comment]
Thanks!
